### PR TITLE
Add image gallery example using turbo-frames & Flickr API

### PR DESCRIPTION
**Pull request by matteo95g**

### CODE REVIEW

File: app/services/flickr_service.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+require 'net/http'
+
+class FlickrService
+  FLICKR_API_URL = 'https://api.flickr.com/services/rest/?method=flickr.photos.search&api_key='.freeze
+
+  def search(query)
+    uri = URI.parse("#{FLICKR_API_URL}#{ENV['FLIKR_API_KEY']}&tags=#{query}&per_page=24&format=json&nojsoncallback=1")
```
Comment (brunvez) [inline app/services/flickr_service.rb]: Can we use [`URI::HTTP`](https://ruby-doc.org/stdlib-2.5.1/libdoc/uri/rdoc/URI/HTTP.html) and [`Hash#to_query`](https://apidock.com/rails/ActiveSupport/CoreExtensions/Hash/to_query)?

File: app/services/flickr_service.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    JSON.parse(Net::HTTP.get(uri))
```
Comment (brunvez) [inline app/services/flickr_service.rb]: Extra points for no external dependencies

File: app/views/gallery/_images.html.erb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<%= turbo_frame_tag 'images' do %>
+  <%= form_with(url: images_gallery_index_path, class: 'flex justify-center mb-5') do |form| %>
```
Comment (brunvez) [inline app/views/gallery/_images.html.erb:2]: Isn't this repeated?

File: app/views/gallery/_images.html.erb [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+    <%= form.button(:submit, class: 'cursor-pointer ml-1 text-white bg-gray-800 rounded px-1') do %>
+      <i class='fas fa-search'></i> Search
+    <% end %>
+  <% end %>
+
+  <div class='grid grid-cols-4 gap-2'>
+    <% images.each do |image| %>
+      <div class='flex justify-center items-center shadow hover:shadow-lg'>
+        <img src=<%= "https://farm#{image['farm']}.staticflickr.com/#{image['server']}/#{image['id']}_#{image['secret']}_m.jpg" %> />
```
Comment (JulianPasquale) [inline app/views/gallery/_images.html.erb]: Can we move this into a helper? or we can also create an Image component and use it here

**Review by JulianPasquale — APPROVED** [review]

**Review by JuannFerrari — APPROVED** [review]
Great job!

**Pull request merged**